Source/santad/BUILD

```diff
@@ -73,6 +73,19 @@ objc_library(
     ],
 )
 
+objc_library(
+    name = "santad_test_lib",
```

Comment:
Are you expecting to add other classes to this library that would be used in several tests? If not, EndpointSecurityTestLib would fit better.